client/src/lib/components/sandbox/play/phase-input/ShootingPhaseInput.svelte

```diff
@@ -0,0 +1,127 @@
+<script lang="ts">
+	import { MinaArenaClient } from '$lib/mina-arena-graphql-client/MinaArenaClient';
+
+	export let game: Game;
+	export let currentPlayer: string;
+	export let rerender: () => {};
```

Comment:
Is this a trick you picked up somewhere?  What's the purpose?

Reply:
I just copied this from `MovementPhaseInput` and it was already there.

When we submit the phase data, it does do this:
```
await minaArenaClient.submitShootingPhase(...);
await rerender();
```
although our `rerender` function here is blank. Does this somehow relate to how it refreshes when it receives the results of the phase submission?